src/providers/twitch/TwitchAccount.hpp

```diff
@@ -105,10 +107,11 @@ class TwitchAccount : public Account
     const bool isAnon_;
     Atomic<QColor> color_;
 
-    mutable std::mutex ignoresMutex_;
     QStringList userstateEmoteSets_;
-    UniqueAccess<std::set<TwitchUser>> ignores_;
-    UniqueAccess<std::set<QString>> ignoresUserIds_;
+
+    ScopedCancellationToken blockToken_;
+    QSet<TwitchUser> ignores_;
+    QSet<QString> ignoresUserIds_;
```

Comment:
Could we make these `std::set/std::unordered_set` instead? That way we don't have to mess with extra hashing for TwitchUser